### PR TITLE
Adding deployment files for contrib and docs repos

### PR DESCRIPTION
**Pull request by foxish**
https://github.com/kubernetes/contrib/issues/1304

The PV, PVC, secret and configmap for each of those repos
Also modifications to the ingress to accommodate the new services.

cc @lavalamp @apelisse @pwittrock

### CODE REVIEW

**Comment by apelisse** [comment]
That looks sane. Do you plan to update the makefile in a different pull-request?

**Comment by foxish** [comment]
@apelisse The makefile has been updated already as part of https://github.com/kubernetes/contrib/pull/1436. @lavalamp and I pushed a new submit-queue yesterday which uses the configmaps to pass parameters. I'm updated https://github.com/kubernetes/contrib/pull/1417, with instructions on how to deploy a new submit-queue.

**Comment by apelisse** [comment]
Cool!

File: mungegithub/submit-queue/deployment/docs/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  config.project: kubernetes.github.io
+  config.pr-mungers: size
+  config.state: open
+  config.token-file: /etc/secret-volume/token
+  gitrepo.repo-dir: /gitrepos
+
+  # test-options feature options.
+  test-options.required-retest-contexts: "\"\""
+
+  # submit-queue options.
```
Comment (lavalamp) [inline mungegithub/submit-queue/deployment/docs/configmap.yaml]: Is there an option to set the labels required? Docs needs both a docs LGTM and a tech LGTM. @pwittrock

Comment (pwittrock) [inline mungegithub/submit-queue/deployment/docs/configmap.yaml]: If not, can we disable the merge-bot munger and keep the assignment munger?

Comment (foxish) [inline mungegithub/submit-queue/deployment/docs/configmap.yaml]: This one only turns on the size munger. We can turn on the assignment munger only after https://github.com/kubernetes/kubernetes.github.io/pull/929 is merged I think.

**Comment by foxish** [comment]
Now that https://github.com/kubernetes/contrib/pull/1438 is merged, we can start up the submit-queue application with blunderbuss for contrib. I've updated the munger list for contrib.

**Comment by apelisse** [comment]
Very cool, can't wait!

**Pull request merged**